src/backend/src/services/teams.services.ts

```diff
@@ -35,6 +35,33 @@ export default class TeamsService {
     return teamTransformer(team);
   }
 
+  /**
+   * Changes the description of the given team to be the new description
+   * @param user The user who is editing the description
+   * @param teamId The id for the team that is being edited
+   * @param newDescription the new description for the team
+   * @returns The team with the new description
+   */
+  static async editDescription(user: User, teamId: string, newDescription: string): Promise<Team> {
```

Comment:
this function should be located in the same order that the function is located in the controller file for clarity